### ISSUE TITLE
Write the code for economic events API in Django env. 

### ISSUE THREAD

**Issue by enozcan** [bounswe/bounswe2019group6#64]
Until 04.05.2019 - 23:59

**Comment by enozcan** [comment]
@irmakguzey I've created a branch named "economic-events" from "django-tutorial". I've also refactored the code we wrote today after the meeting and imported it to the project. Can you check if I missed any point in the code?

**Comment by enozcan** [comment]
Moreover, without being a premium user on the website we found, only small amount of data can be fetched. I've requested full access from the owners, but if you encounter a better service for API we can use it instead.

**Comment by irmakguzey** [comment]
Hey @enozcan I checked the code everything looks in place. Thanks.
And I believe that fetching a limited number of events every day will not be a major problem for us. Possible limitation of it is not to be able filter the data thoroughly, which in my opinion is not really necessary. I think we can still reflect the fluctuations with a limited number of events as well. 

And I have updated the pattern paths in urls.py. And added the name of our app to settings.py. A review and a feedback is appreciated.
With these last two modifications our app should be embedded to the system.
